src/applications/ivc-champva/shared/components/applicantLists/ApplicantRelationshipPage.jsx

```diff
@@ -275,11 +275,12 @@ export default function ApplicantRelationshipPage({
           required
           error={checkError}
           onVaValueChange={handlers.radioUpdate}
+          name={`root_${keyname}`}
         >
           {options.map(option => (
             <va-radio-option
               key={option.value}
-              name="describes-you"
+              name={`root_${keyname}`}
```

Comment:
Have to add this dynamic `name` property so that Cypress can pick up on the input fields